src/st_pages/__init__.py

```diff
@@ -105,10 +105,10 @@ def _add_page_title(
 add_page_title = _gather_metrics("st_pages.add_page_title", _add_page_title)
 
 
-@cache_resource
+# @cache_resource
```

Comment:
This should be put back, since otherwise the emojis will be re-loaded every time.